nucleus/parent/pom.xml

```diff
@@ -1074,7 +1074,7 @@
                         <dateFormatTimeZone>UTC</dateFormatTimeZone>
                         <commitIdGenerationMode>full</commitIdGenerationMode>
                         <includeOnlyProperties>
-                            <includeOnlyProperty>git.branch</includeOnlyProperty>
+                            <!--includeOnlyProperty>git.branch</includeOnlyProperty--><!-- not reproducible -->
```

Comment:
If this isn't working for reproducibility - is the `git.closest.tag.commit.count` below?